spec/controllers/idv/personal_key_controller_spec.rb

```diff
@@ -181,6 +195,15 @@ def index
           expect(response).to redirect_to idv_come_back_later_path
         end
       end
+      it 'logs key submitted event' do
+        patch :update
+
+        expect(@analytics).to have_logged_event(
+          'IdV: personal key submitted', address_verification_method: nil,
+                                         deactivation_reason: 'gpo_verification_pending',
+                                         proofing_components: nil
```

Comment:
style nit, my approach for these things is "all same line or all different lines" so I'd do:
```suggestion
          'IdV: personal key submitted',
          address_verification_method: nil,
          deactivation_reason: 'gpo_verification_pending',
          proofing_components: nil,
```